packages/rest/src/rest.server.ts

```diff
@@ -582,11 +584,15 @@ export class RestServer extends Context implements Server, HttpServerLike {
     const protocol = await this.get<'http' | 'https' | undefined>(
       RestBindings.PROTOCOL,
     );
+    const key = await this.get<string | undefined>(RestBindings.KEY);
+    const cert = await this.get<string | undefined>(RestBindings.CERT);
 
     this._httpServer = new HttpServer(this.requestHandler, {
       port: port,
       host: host,
       protocol: protocol || 'http',
+      key: key,
+      cert: cert,
```

Comment:
Please rework this code to support PFX-based configuration and also additional TLS config options.